web/src/admin/admin-overview/AdminOverviewPage.ts

```diff
@@ -33,35 +33,29 @@ export function versionFamily(): string {
     return parts.join(".");
 }
 
+const customCSS: Readonly<CSSResult> = css`
```

Comment:
Common change type two: custom CSS, so lift it out of the class, make it `Readonly<CSSResult>`, and put the name in-line, creating a more readable `style()` declaration.